pallets/auctions/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+edition = "2018"
```

Comment:
I just realized that I didn't merge my Cargo.toml changes.
The correct order is: name, version, edition, license